src/ocaml-output/Makefile

```diff
@@ -89,7 +96,6 @@ endif
 	                    -e '/%type.*->.*/d' \
 	                    >> $@
 
-.PHONY: fstar.exe
```

Comment:
We just moved up the `.PHONY` target a little above in the file (and put the ocaml printer as phony since I don't know a good way to have the correct version of the file (depending of the current ocaml version) as a dependency)